stylesheets/abstract/variables.css

```diff
@@ -14,4 +14,11 @@
 
   --bg-blue: rgb(25, 0, 255);
   --badge: rgb(253, 143, 0);
+
+  --grid-item1: rgb(255, 69, 0);
+  --grid-item2: rgb(154, 205, 50);
+  --grid-item3: rgb(106, 90, 205);
+  --grid-item4: rgb(225, 105, 180);
+  --grid-item5: rgb(65, 105, 225);
+  --grid-item6: rgb(218, 165, 32);
```

Comment:
you can use named colors so that it is easier for you to debug in case something happens and for viewer as well. i am not saying to add variables, just normal named colors are fine. again, its up to your preference.

Reply:
okay as u suggested